src/js/structs/Node.js

```diff
@@ -83,6 +83,10 @@ class Node extends Item {
 
     return sum;
   }
+
+  getResources() {
```

Comment:
Let's please rename it to `getUsedResources` since we do have `this.get("resources")`

Reply:
@nLight the reason I left as `getResource` and not `getUsedResources` is that in `structs/Framework.js` we already use `getResources` in which is used here https://github.com/dcos/dcos-ui/pull/2791/files#diff-def918eb22daa7712f7a5e81753e82ebR51

So to keep it consistent and easier to know what to expect I kept `getResources`, Now I wonder. Should I change this method to be the same as in `Framework.js` or introduce `getUsedResources` for framework and node then change line `https://github.com/dcos/dcos-ui/pull/2791/files#diff-def918eb22daa7712f7a5e81753e82ebR51` to get `getUsedResources`

Thanks for the feedback

Reply:
@weblancaster while I understand the path you chose to prevent further code changes, this is a good opportunity to address the name issue here and name it correctly.

Reply:
Comments addressed